tests/system/BaseTestConfig.ts

```diff
@@ -0,0 +1,7 @@
+class BaseTestConfig {
```

Comment:
I am not sure that class is needed here, simple object is good enough.
And what is the reason to use snake case here? and not camel case which we use everywhere?

Reply:
ty! It's my fault (python code wants to get out 😅) I will fix it

Reply:
Agree with Alexander